peopleapi/Program.cs

```diff
@@ -1,24 +1,26 @@
-﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Hosting;
+using Serilog;
 
-namespace peopleapi
-{
-    public class Program
-    {
-        public static void Main(string[] args)
-        {
-            CreateWebHostBuilder(args).Build().Run();
+namespace peopleapi {
+    public class Program {
+        public static void Main(string[] args) {
+            var logger = new LoggerConfiguration()
+                .MinimumLevel.Information()
+                .WriteTo.Console()
+                .CreateLogger();
+
+            CreateHostBuilder(args)
+                .UseSerilog(logger)
+                .Build()
+                .Run();

```

Comment:
Serilog setup procedure. This creates a console logger. You'll have to look into Serilog's LogFile extension to see how to add logging to text files - though the issue will also be setting up a docker volume to persist those files to disk.